README.md

```diff
@@ -2151,10 +2151,10 @@ no parameters.
   ```
 
 
-* <a name="camelcase-classes"></a>
-  Use `CamelCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
+* <a name="pascalcase-classes"></a>
```

Comment:
I guess the anchor should not change immoderately. So, I would like to still use `camelcase-classes` as an anchor, and use `Upper Camel Case` instead of `Pascal Case`. WDYT?

Reply:
Good point! Let's keep the name for now.